packages/core/src/theme/colors.ts

```diff
@@ -29,7 +29,7 @@ const shadeRed = '#6d0000'
 // orange
 const lightOrange = '#fef2e7'
 const tintOrange = '#faaf69'
-const orange = '#f68013'
+const orange = '#f06f20'
```

Comment:
This is the only change. Changing the color of orange. Everything else is snapshots 📸